client/src/pages/spectrogram/components/PluginsPane.tsx

```diff
@@ -94,7 +100,11 @@ export const PluginsPane = ({ cursorsEnabled, handleProcessTime, meta, setMeta }
           toast.error(`Plugin failed to run: ${data.status}`);
           return;
         }
-        if (data.data_output && data.data_output.length > 0) {
+        if (
+          data.data_output[0]['data_type'] == MimeTypes.IQ_SIGMF_32 &&
```

Comment:
I would move to after checking data.data_output is null and data.data_output.length>0

Reply:
good catch